src/api.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const fetchArticles = () => {
+  return axios
+    .get("https://nc-news-npk0.onrender.com/api/articles")
+    .then((response) => {
+      return response.data.articles;
+    })
+    .catch((error) => {
```

Comment:
Rarther than catching any errors here and console logging them, I'd reccomend having a catch block in your articles component instead so you can use any errror to display a message to the user.